README.md

```diff
@@ -75,20 +73,58 @@ $ cd kyt-cli
 $ go run main.go --server "http://localhost:8080/v1" dlm get devices
 ```
 
-## Repo Notes
+## Build with CI/CD
 
-### OpenAPI specification
+[Concourse CI](https://concourse-ci.org/) will be used as CI/CD system.
 
-The REST-API is defined with the OpenAPI document `kyt-api-spec/kytapi.yaml`. From this specification, parts of the server and client go code are automatically generated.
+Download `fly` and login to concourse CI server.
+
+```bash
+# Download fly from concourse server
+$ sudo curl -L https://concourse.ci4rail.com/api/v1/cli?arch=amd64&platform=linux -o /usr/local/bin/fly && sudo chmod +x /usr/local/bin/fly
+# Login to concourse
+$ fly -t prod login -c https://concourse.ci4rail.com
+```
+## pipeline.yaml
+
+The `pipeline.yaml` is the CI/CD pipeline that builds kyt-cli called `kyt` and kyt-api-server docker image. The kyt-cli goes to a [gitub pre-release](https://github.com/ci4rail/kyt-cli/releases) and the `kyt-api-server` will be published as docker image [here](https://harbor.ci4rail.com/harbor/projects/7/repositories/kyt%2Fkyt-api-server).
+
+### Usage
+
+Copy `ci/credentials.template.yaml` to `ci/credentials.yaml` and enter the credentials needed. The `github_access_token` needs `write:packages` rights on Github.
+Apply the CI/CD pipeline to Concourse CI using
+```bash
+$ fly -t prod set-pipeline -p kyt-services -c pipeline.yaml -l ci/config.yaml  -l ci/credentials.yaml
+```
+
+## pipeline-pullrequests.yaml
+
+The `pipeline-pullrequests.yaml` is the CI/CD pipeline that checks Github for commits within Pull Requests and performs a clean build of kyt-cli `kyt` and `kyt-api-server` go binaries. It also runs `go test` for both.
```

Comment:
Better: "The `pipeline-pullrequests.yaml` defines a pipeline that runs basic quality checks for each new pull request. For this, consourse checks Github for new pull requests. If a new pull request is found, it downloads the repo, performs a clean build of kyt-cli `kyt` and `kyt-api-server` go binaries. It also runs `go test` for both.

Reply:
It does not only look for new PRs. It also looks for new PRs, but it also looks for commits in previously active PRs. 
Will rephrase it to:
The `pipeline-pullrequests.yaml` defines a pipeline that runs basic quality checks on pull requests. For this, consourse checks Github for new or changed pull requests If a change is found, it downloads the branch and performs a clean build of kyt-cli `kyt` and `kyt-api-server` go binaries. It also runs go test for both.